### DISCUSSION TITLE
Storage Size of Loaded Data compared to other graph databases

### DISCUSSION THREAD

**Discussion by annainfo** [terminusdb/terminusdb#1660] [Q&A]
https://www.tigergraph.com/blog/tigergraph-roars-past-the-competition-graph-database-benchmark-findings/

1) "TigerGraph reduced the input data size by 50% for Graph 500 dataset and by 61% for the Twitter dataset."
2) "The other graph databases demanded 5x to 13x more storage space than TigerGraph."

Graphic at
https://www.embedded-software-engineering.de/neo4j-amazon-neptune-und-tigergraph-graph-datenbanken-im-vergleich-gal-957899/?p=2#gallerydetail

How much "Storage Size of Loaded Data" does TerminusDB have for these datasets?

**Comment by GavinMendelGleason** [comment]
Thanks for this question - we haven't loaded these datasets, but we are likely to be very competitive with TigerGraph. We'll take a look.

**Reply by annainfo** [reply]
https://www.tigergraph.com/benchmark/ 
1) at the bottom has the 2018 test with a 2-page-PDF for download w/o registration
2) in the middle there is "Third Party Benchmark: Neo4j vs TigerGraph July 2019 In-Depth Benchmarking of Graph Database Systems using the Linked Data Benchmark Council (LDBC) Social Network Benchmark (SNB).", summary at https://cdn2.hubspot.net/hubfs/4114546/LDBC_brief_benchmark.pdf full article at https://arxiv.org/abs/1907.07405 - page 11 has "Loaded data size"
